tests/sentry/mediators/sentry_app_installations/test_installation_notifier.py

```diff
@@ -34,24 +43,35 @@ def test_task_enqueued(self, safe_urlopen):
             user=self.user,
         )
 
-        safe_urlopen.assert_called_once_with(
-            'https://example.com',
-            json={
-                'action': 'installation',
-                'installation': {
-                    'uuid': self.install.uuid,
+        data = faux(safe_urlopen).kwargs['data']
+
+        assert data == {
+            'action': 'created',
+            'installation': {
+                'uuid': self.install.uuid,
+            },
+            'data': {
+                'app': {
+                    'uuid': self.sentry_app.uuid,
+                    'slug': self.sentry_app.slug,
                 },
-                'data': {
-                    'app': {
-                        'uuid': self.sentry_app.uuid,
-                        'slug': self.sentry_app.slug,
-                    },
-                    'organization': {
-                        'slug': self.org.slug,
-                    },
-                    'uuid': self.install.uuid,
-                    'code': self.install.api_grant.code,
+                'organization': {
+                    'slug': self.org.slug,
                 },
+                'uuid': self.install.uuid,
+                'code': self.install.api_grant.code,
             },
-            timeout=5,
-        )
+            'actor': {
+                'id': self.user.id,
+                'name': self.user.name,
+                'type': 'user',
+            },
+        }
+
+        assert faux(safe_urlopen).kwarg_equals('headers', DictContaining(
+            'Content-Type',
+            'Request-ID',
+            'Sentry-Hook-Event',
+            'Sentry-Hook-Timestamp',
+            'Sentry-Hook-Signature',
```

Comment:
You should probably also have a test that ensures the signature can be verified for a request as customers will likely implement that and we wouldn't want to break it by accident.